### ISSUE TITLE
[Bug]: Regular looking Bold in VS Code

### ISSUE THREAD

**Issue by dinhquochan** [MonoLisaFont/feedback#214]
### Edition

Plus

### What version are you using?

2.007

### What type of weights are you using?

Default (non-customized)

### Operating system

Windows 11

### Program

VS Code

### Display resolution

2560x1440

### What happened?

Before 2.007, it's ok. I don't know why. I see the old issue #31 that tells me I should use TFF version, v2 is TFF already.
I attach the screenshot, 1 JetBrains (left) 1 VS Code (right), Font 14px _ General, Turn off ligature:

![image](https://github.com/MonoLisaFont/feedback/assets/9979458/0a430385-2334-4d93-99bd-27fe6ba772ed)

**Comment by bebraw** [comment]
Can you add your VS Code configuration? Which weight are you using there?

**Comment by dinhquochan** [comment]
Yes, here:

```json
{
    "editor.fontFamily": "MonoLisa",
    "editor.fontLigatures": false,
    "editor.fontSize": 14,
    "editor.fontWeight": "400"
}
```

**Comment by bebraw** [comment]
When using a number value, you should use `"editor.fontWeight": 400,`. You can also refer to weights by their name (i.e., `"editor.fontWeight": "bold",`).

**Comment by dinhquochan** [comment]
@bebraw It's ok before v2.007, not is string or number, you can see, I see from 100->500 is not affected, only 600 is bold, before it should be from light->regular->medium->bold. I will show you the old version.

![image](https://github.com/MonoLisaFont/feedback/assets/9979458/5147ec84-9bf9-4b56-b628-098c500ccfc9)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/a0668838-7d16-4e71-9b1c-78d525a9f433)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/bbe2a3b8-ddcc-45a8-8e6d-0e18e6c8d1de)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/bb4d1f9b-607a-44be-b1f1-5747db9b9957)

**Comment by dinhquochan** [comment]
Here is in v2.006 @bebraw 

![image](https://github.com/MonoLisaFont/feedback/assets/9979458/d97b6936-ea90-484d-99ec-11a6cd2c4f05)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/80edb941-9406-4dc0-b05d-0bdb03ce66d8)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/eb4e8218-5903-4f20-bce1-94272800a208)
![image](https://github.com/MonoLisaFont/feedback/assets/9979458/91ab23ea-c682-406c-abde-7e19e4257b5c)

**Comment by bebraw** [comment]
I tested this using Plus (2.007) on MacOS and scaling works perfectly there when using a number for `fontWeight` as instructed by VS Code.

You could also check with https://fontdrop.info/ and compare the files for 2.006 and 2.007.

I guess we would need someone using Windows to understand the issue better.

**Comment by dinhquochan** [comment]
Yes. On my Mac it is ok too. Only windows 🥲

**Comment by bebraw** [comment]
Ah, thanks for confirming. There's another Windows related issue so it feels like it's something coming from the build/api.

As an intermediate fix we might do a build through the old process. My gut feeling is that some Windows specific metadata might be off somehow.

**Comment by bebraw** [comment]
@dinhquochan We have a potential fix available now. Can you head to https://www.monolisa.dev/orders and choose **prerelease** before downloading and testing?

<img width="520" alt="Screenshot 2023-05-12 at 12 28 38" src="https://github.com/MonoLisaFont/feedback/assets/166921/8ffa9bf3-c6b0-4d32-a919-1eb50f55dc1e">

**Comment by dinhquochan** [comment]
Hi @bebraw, it's fixed on pre-released.

![image](https://github.com/MonoLisaFont/feedback/assets/9979458/2168b2bb-e8a0-4ef7-bd6c-2a48d9d76dea)

**Comment by bebraw** [comment]
Published as 2.008. Thanks for the report.
